nn/normalization.py

```diff
@@ -183,3 +184,35 @@ def normalize(a: nn.Tensor, *, axis: Union[nn.Dim, Sequence[nn.Dim]], epsilon: f
 
   mean, variance = nn.moments(a, axis=axis)
   return (a - mean) * nn.rsqrt(variance + epsilon)
+
+
+class Normalize(nn.Module):
+  """
+  :func:`normalize` with additional scale and bias
+  """
+
+  def __init__(self,
+               param_shape: Sequence[nn.Dim],
+               epsilon: float = 1e-6,
+               scale: bool = True, bias: bool = True):
+    """
+
```

Comment:
There should be no empty line here (unless you have some initial description, which you do not have here).